src/core/server/saved_objects/import/lib/validate_references.ts

```diff
@@ -25,19 +26,26 @@ const getObjectsToSkip = (retries: SavedObjectsImportRetry[] = []) =>
 export async function getNonExistingReferenceAsKeys(
   savedObjects: SavedObject[],
   savedObjectsClient: SavedObjectsClientContract,
-  namespace?: string,
+  namespace: string | undefined,
+  importStateMap: ImportStateMap,
   retries?: SavedObjectsImportRetry[]
```

Comment:
Optional: given the increasing number of parameter for this function, switching to an object parameters would probably helps with the readability

```ts
function getNonExistingReferenceAsKeys({
  savedObjects, 
  savedObjectsClient, 
  namespace, 
  importStateMap, 
  retries
}: {
  savedObjects: SavedObject[],
  savedObjectsClient: SavedObjectsClientContract,
  namespace?: string,
  importStateMap: ImportStateMap,
  retries?: SavedObjectsImportRetry[]
})
```

Reply:
Done in 662b306cd2c9671b793d3cda12dbb808526e0b1e